test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -241,6 +241,52 @@ function prepare-log-file {
 	chown root:root $1
 }
 
+# A helper function for removing salt configuration and comments from a file.
+# This is mainly for preparing a manifest file.
+#
+# $1: Full path of the file to manipulate
+function remove-salt-config-comments {
+	# Remove salt configuration.
+	sed -i "/^[ |\t]*{[#|%]/d" $1
+	# Remove comments.
+	sed -i "/^[ |\t]*#/d" $1
+}
+
+# Prepare the abac policy file from the salt template.
+function setup-abac-authz-policy-file {
```

Comment:
Thank you for pointing this out @deads2k . There seem to be no parts of kubemark that are used with ABAC as Wojtek said. However I just let it stay from PR [#38626](https://github.com/kubernetes/kubernetes/pull/38626) thinking that ABAC works as a fallback option in the case RBAC fails. Removed it now.
However, it doesn't work right now, as expected, because we need some ClusterRoleBindings for kubemark as well (similar to the addons/e2e-rbac-bindings you've suggested). For now I did a weird hack by using kubectl with kubelet credentials. However we have to add these role-access pairs for kubemark to work ( @wojtek-t @gmarek do they look correct and complete?):
ROLE NAME                              ACCESS LEVEL             TO BE USED BY
kubecfg-cluster-admin                   view                                kubectl (currently only the local machine running start-kubemark.sh uses this and all it does is get nodes in kubemark cluster)
kubelet-cluster-admin                    cluster_admin                 hollowKubelet (also similar for node-problem-detector maybe, since it does PATCH?)
admin-cluster-admin                      cluster_admin                controller-manager

To add further, looking at the apiserver log RBAC DENY messages, we know that 'kubelet' should be able to 'list endpoints' and 'kubecfg' should be able to 'list endpoints' and 'list nodes'. Once we ensure that we give these access rights, we can go ahead and run-e2e-tests script and check if we still have any RBAC DENY msgs.